vscode/quint-vscode/scripts/prepare-release.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+QUINT_VERSION=0.5.2
```

Comment:
can we take this from an env variable? or is there a reason we want it hard coded in the script?

Reply:
I don't really know bash, so I didn't want to go to the trouble of figuring out how to handle undefined parameters at this point, so I just used the simplest approach I could think of. But this should be easy enough to copy past from stack overflow, I'll look it up.

Reply:
at the top of the script, you can set these flags to handle undefined params and other common errors automatically:

```
set -euo pipefail

set -o xtrace
```

but a sanity check to report an error if this is unset probably also makes sense